DRAFT_Connectivity.Rmd

```diff
@@ -1,266 +1,280 @@
+7.	You export the results as shapefiles for both scenarios.
+8.	The scaled values for the "Connectivity" indicator are determined by comparing two scenarios: one that includes the presence of infrastructure and one that excludes infrastructure. The reference value is calculated as the ratio of the mean distances between myrpolygon areas in these two scenarios.
+
+Currently step 8, and a subsequent step 9 (to aggregate scaled indicators to regions and simultaneously calculate errors) are not complete. Se [this issue]()https://github.com/NINAnor/ecosystemCondition/issues/144. 
+
+It is also possible to create a time series for this indicator, but this is not yet done. See [GitHub issue](https://github.com/NINAnor/ecosystemCondition/issues/145).
+
+## About the underlying data {#underlying-data-con}
+The **infrastructure index** is one part of the connectivity index for wetlands. The infrastructure index was calculated for three time points: 2003, 2013, and 2023. This provides a temporal perspective for assessing changes over time. The data used included information on various types of infrastructure, such as roads and buildings. Additionally, a **new wetland** map for Norway has been developed. This map is based on Sentinel-2 and LiDAR data from the year 2020. The U-Net model, implemented through Google Earth Engine and TensorFlow, has been used for mapping wetlands in southern Norway. The model achieved a balanced accuracy rate of 90.9% when validated against ground-truth samples, significantly improving upon the accuracy rates achieved with manually digitized maps in Norway. 
+
+### Representativity in time and space {#representativity-con}
+The data for the infrastructure index covers the entire country and was collected for three time points over a 20-year period. This provides a broad and representative dataset for assessing changes in connectivity in Norwegian landscapes. The wetland mapping was conducted using data from the year 2020, making it representative for that specific time frame. However, the model used in the mapping process can be applied to other time frames if necessary. 
+
+### Original units {#units-con}
+The infrastructure index has no specific units as it is a measure of landscape fragmentation and disruption. It is expressed as a dimensionless index.
+
+### Temporal coverage {#temporal-coverage-con}
+The data covers three time points: 2003, 2013, and 2023. This provides a 20-year temporal perspective for assessing changes in connectivity over time. The wetland mapping provides information in the form of pixel values representing wetland presence or absence.
+
+### Aditional comments about the dataset {#additional-comments-con}
+It is important to note that infrastructure data are based on available map and geographical data sources. Sometimes, there may be variations and uncertainties in this data that can affect the accuracy of the infrastructure index.
+
+## Ecosystem characteristic {#egenskap-con}
```

Comment:
I changed the **økologisk egenskap** and **ECT** to landscape ecological effects. This has been the tradition and I think it a very goo match with these categories.